liblangutil/Scanner.cpp

```diff
@@ -321,16 +324,18 @@ int Scanner::scanSingleLineDocComment()
 		{
 			// Check if next line is also a single-line comment.
 			// If any whitespaces were skipped, use source position before.
-			if (!skipWhitespace())
+			if (!skipWhitespaceExceptUnicodeLinebreak())
```

Comment:
This fixes the issue where a line break between two natspec comments will mean end of block for the first one. For e.g., the following is two seperate natspec comments.

```
\\\ hello

\\\ world
```